src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    String name;
+    int year;
```

Comment:
why do you recreate fields that already exist at the parent?

Reply:
didnt guess.done.